contrib/linux/dosbox-staging.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=dosbox-staging
+GenericName=DOS emulator
+Comment=A modernized DOSBox
+Exec=dosbox
```

Comment:
(outside of the scope of this review, just starting dicussion)

I think we'll need to rename the binary finally… Otherwise, our package will clash with official DOSBox package and it will be a problem not only to users, but also for development.

@krcroft What do you think? Is it time to rename binary? Do we want `dosbox-staging` or something else? (`dos`?)

Reply:
My 2 cents, I think having both DOSBox and dosbox-staging installed isn't really useful, so I would prefer it this way. Packages just need to make sure they are set up correctly to make the package manager able to inform the user that they can't have both installed.